apps/api/src/app/workflows-v2/usecases/build-variable-schema/build-available-variable-schema.usecase.ts

```diff
@@ -44,9 +45,10 @@ export class BuildAvailableVariableSchemaUsecase {
     } as const satisfies JSONSchemaDto;
   }
 }
+
 function safePayloadSchema(workflow: NotificationTemplateEntity): JSONSchemaDto | undefined {
```

Comment:
`For type safety, should we return undefined, null or {}? What's more convenient? I'd probably go with null.`

I also prefer using `null`. Returning `{}` is unpredictable and reveals issues at runtime. We need to find a reliable way to avoid using empty objects.